ConsoleGame.java

```diff
@@ -63,12 +80,10 @@ public void displayAllGameModes() {
     }
 
     @Override
-    public void gameIsOver(Board board, Player winner) {
-        if (!board.checkForWin('x') && !board.checkForWin('o')) {
-            print("Draw!\n");
-            return;
+    public void gameIsOver(Board board) {
+        if (board.isGameOver()) {
+            print(board.getWinner() == null ? "Draw! \n" : board.getWinner() + " has won the game! \n");
```

Comment:
This ternary could be pulled out into a method e.g. announceResult() or something.